sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for \
+easy large numeric calculations, but which also can be easily programmed for \
+difficult or long calculations. It can accept a command line \
+argument, in which case it executes that single command and exits. \
+Otherwise, it enters interactive mode. In this mode, it accepts \
+commands one at a time, processes them, and displays the answers. \
+In the simplest case, commands are simply expressions which are evaluated."
```

Comment:
In order to follow the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines), could you please change DESCRIPTION to:
```
DESCRIPTION="Calc is an interactive calculator which provides for easy large \
numeric calculations, but which also can be easily programmed for difficult or \
long calculations. It can accept a command line argument, in which case it \
executes that single command and exits. Otherwise, it enters interactive mode. \
In this mode, it accepts commands one at a time, processes them, and displays \
the answers. In the simplest case, commands are simply expressions which are \
evaluated."
```
(The text is unchanged. I've only moved the line breaks, to make all lines as long as possible, but still <= 80 chars long.)